gallery/src/components/NxTable/NxTablePaginationExample.tsx

```diff
@@ -22,7 +22,13 @@ const tableData: Row[] = [
   {name: 'Karen', country: 'Australia'},
   {name: 'Paul', country: 'UK'},
   {name: 'Raul', country: 'Argentina'},
-  {name: 'Maria', country: 'Spain'}
+  {name: 'Maria', country: 'Spain'},
```

Comment:
Was getting a cannot have duplicate aria-label errors for the pagination labels. Both the pagination and the pagination and filtering example rendered 2 pages, so both of the examples had "pagination, page 1 of 2". I added more rows to get around the error.

Reply:
> Was getting something similar to a "cannot have duplicate aria-label" errors for the pagination labels. Both the pagination and the pagination and filtering example rendered 2 pages, so both of the examples had "pagination, page 1 of 2". To get around the error, I added more rows so that the pagination examples had 3 pages and the labels were different.

That sounds like a legitimate problem though and working around it just in this specific case doesn't really solve it.

The real issue here is that the pagination is a `<nav>`, which gives it a role of `navigation` by default, and that's a landmark role.  All landmarks of the same role must be distinguishable by accessible name.  I don't think pagination should be a landmark, so we probably need to change either the `role` or the element tagname.  I'm not sure which would be better to change.  Looking through the history, it seems that it was originally a `<div>` and was changed to a `<nav>` during some a11y-minded work.  I'm not sure though if that change actually made some difference or was just a "this seems right" kind of thing.

Reply:
> The real issue here is that the pagination is a <nav>, which gives it a role of navigation by default, and that's a landmark role. All landmarks of the same role must be distinguishable by accessible name. I don't think pagination should be a landmark, so we probably need to change either the role or the element tagname. I'm not sure which would be better to change. Looking through the history, it seems that it was originally a <div> and was changed to a <nav> during some a11y-minded work. I'm not sure though if that change actually made some difference or was just a "this seems right" kind of thing.

Hmm, I just checked how google does it in their search results and they use `<div role="navigation">`.  But in their case, the navigation is more of a global thing, the search results are the whole page that that control goes to the next page.  `NxPagination` might be used in multiple situations, some like that and some not so much (like when it pages a table that's just part of the page).

Reply:
> Hmm, I just checked how google does it in their search results and they use <div role="navigation">. But in their case, the navigation is more of a global thing, the search results are the whole page that that control goes to the next page. NxPagination might be used in multiple situations, some like that and some not so much (like when it pages a table that's just part of the page).

`<nav>` supports the following aria roles:
* `navigation`: what it already has.  Not necessarily the best, as it makes it a landmark and also the descriptions of this role generally talk about links, like if each pagination item actually navigated the page URL, which isn't the case for us
* `menu`: seems too vague
* `menubar`: a horizontal menu akin to the File, Edit, etc dropdowns across the top of a desktop application.  Also doesn't seem right for us
* `none`/`presentation`: removes from screenreader support, definitely not what we want
* `tablist`: tabs, also not what we want.  

None of these seem to quite match our use case.  Maybe we need to do more research on what role this should have, possibly moving away from it being a `<nav>`

Reply:
> > Hmm, I just checked how google does it in their search results and they use . But in their case, the navigation is more of a global thing, the search results are the whole page that that control goes to the next page. NxPagination might be used in multiple situations, some like that and some not so much (like when it pages a table that's just part of the page).
> 
> `<nav>` supports the following aria roles:
> 
> * `navigation`: what it already has.  Not necessarily the best, as it makes it a landmark and also the descriptions of this role generally talk about links, like if each pagination item actually navigated the page URL, which isn't the case for us
> * `menu`: seems too vague
> * `menubar`: a horizontal menu akin to the File, Edit, etc dropdowns across the top of a desktop application.  Also doesn't seem right for us
> * `none`/`presentation`: removes from screenreader support, definitely not what we want
> * `tablist`: tabs, also not what we want.
> 
> None of these seem to quite match our use case. Maybe we need to do more research on what role this should have, possibly moving away from it being a `<nav>`


With the new pagination changes, only needed to modify to have unique aria-label: fc6a1260e6e0023df491595ad992cf20f17ec